backend/src/database/models/organization.ts

```diff
@@ -88,6 +88,46 @@ export default (sequelize) => {
         defaultValue: false,
         allowNull: false,
       },
+      founded: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+      },
+      industry: {
+        type: DataTypes.TEXT,
+        allowNull: true,
+      },
+      size: {
```

Comment:
What is this? And how is it different from `employees`? If it's semantically the same (or very similar), we should use employeeCount.

Reply:
`employees` is the number of workers in the company. `size` is the well, size of the company that's small, medium, large, etc. Size is represented as a range.